gcs/sample/gcs-sample.yaml

```diff
@@ -0,0 +1,124 @@
+apiVersion: triggers.tekton.dev/v1alpha1
+kind: TriggerTemplate
+metadata:
+  name: gcs-triggertemplate
+spec:
+  params:
+    - name: pvc-name
+    - name: project-name
+    - name: secret-name
+  resourceTemplates:
```

Comment:
this should actually be `resourcetemplates` - long story regarding how we got away with this but working with @dibyom  it looks like this might be the root of https://github.com/tektoncd/triggers/issues/526

Reply:
😅

Reply:
👍 replaced `resourceTemplates` with `resourcetemplates`